roles/hosted_engine_setup/tasks/alter_libvirt_default_net_configuration.yml

```diff
@@ -6,21 +6,37 @@
   register: default_net_xml
 - name: IPv6 configuration
   block:
+    - name: Accept IPv6 Router Advertisements
+      shell: echo 2 > /proc/sys/net/ipv6/conf/ovirtmgmt/accept_ra
     - name: Remove IPv4 configuration
       xml:
         xmlstring: "{{ default_net_xml.get_xml }}"
         xpath: /network/ip
         state: absent
       register: editednet_noipv4
-    - name: Configure it as an isolated network
+    - name: Configure libvirt default network as an forward network
```

Comment:
nit: "_a_ foward" (not "an"). Typo was there before your patch, but you can fix it :-)

Reply:
7b0a59f includes the fix for the typo